templates/d/src/tic80.d

```diff
@@ -63,12 +63,12 @@ void exit();
 void elli(int x, int y, int a, int b, int color);
 void ellib(int x, int y, int a, int b, int color);
 bool fget(int id, ubyte flag);
-int font(char* text, int x, int y, ubyte transcolors, int colorcount, int width, int height, bool fixed, int scale);
+int font(const char* text, int x, int y, uint* transcolors, int colorcount, int width, int height, bool fixed, int scale, bool alt);
```

Comment:
Are you SURE this is cont char*?

Reply:
Not really to be honest, I've never really done much with D. I added `const` because 1. that's what `print` and `trace` use and 2. it made my test compile.

I *think* `const` here just means the function won't be able to change the value, not that a constant value has to be passed.